sdk/com.ibm.sbt.web/src/main/webapp/js/sdk/_bridges/dojo-amd/i18n.js

```diff
@@ -17,22 +17,10 @@
 /**
  * Social Business Toolkit SDK.
  */
-define(['dojo/i18n', 'dojo/date/locale'],function(i18n, dateLocale) {
+define(['dojo/i18n'],function(i18n) {
 	    var load = function(id, require, callback){	    	
 	    	i18n.load(id, require, callback); 
 	    };
```

Comment:
Few Comments - 
1. Shouldnt we need to make similar changes in the _bridges/dojo/i18n module to remove the extra dependency on dojo/date/locale ? Also in that file we can remove dojo,require statements when we have the amdcompat js. 
2. ConnectionsGridRenderer and TemplatedGridRenderer also has reference to i18n.getUpdatedLabel. They also need to be changed